relay/prover.go

```diff
@@ -379,6 +449,61 @@ func (pr *Prover) ProveHostConsensusState(ctx core.QueryContext, height exported
 	return pr.originProver.ProveHostConsensusState(ctx, height, consensusState)
 }
 
+// shouldUseSHFUGRPC determines whether to use SHFU gRPC server based on config and environment variable
+// Environment variable YRLY_LCP_SHFU_GRPC_ENABLE=yes enables gRPC, disabled by default
+func (pr *Prover) shouldUseSHFUGRPC() (bool, string) {
+	// First check if address is configured
+	if pr.config.ShfuGrpcAddress == "" {
+		// No address configured, cannot use gRPC
+		return false, ""
+	}
+
+	// Check if gRPC is enabled via environment variable
+	envEnable := os.Getenv("YRLY_LCP_SHFU_GRPC_ENABLE")
+	if envEnable != "yes" {
```

Comment:
`strconv.ParseBool` only accepts 1, t, T, TRUE, true, True, 0, f, F, FALSE, false, and False. Given that, I think it makes sense to use "true" instead of "yes" or use the `strconv.ParseBool` function:

```suggestion
	envEnable, _ := strconv.ParseBool(os.Getenv("YRLY_LCP_SHFU_GRPC_ENABLE"))
	if !envEnable {
```

Additionally, the `YRLY_ENABLE_TELEMETRY` environment variable of yui-relayer uses `viper.GetBool`, which calls `strconv.PaseBool`.

Reply:
fixed